account_multicompany_easy_creation/wizards/multicompany_easy_creation.py

```diff
@@ -0,0 +1,374 @@
+        "will overwrite default taxes, but not founded will remain",
+    )
+    # ACCOUNTS
+    smart_search_specific_account = fields.Boolean(
+        default=True,
+        help="Go over specific accounts in actual company to match and set "
+        "equivalent taxes in the new company.\n"
+        "Applies to products, categories, partners, ...",
+    )
+    smart_search_fiscal_position = fields.Boolean(
+        default=True,
+        help="Go over partner fiscal positions in actual company to match "
+        "and set equivalent fiscal positions in the new company.",
+    )
+    update_default_accounts = fields.Boolean(
+        help="Update default accounts defined in account chart template",
+    )
+    account_receivable_id = fields.Many2one(
+        comodel_name="account.account.template", string="Default Receivable Account",
+    )
+    account_payable_id = fields.Many2one(
+        comodel_name="account.account.template", string="Default Payable Account",
+    )
+    account_income_categ_id = fields.Many2one(
+        comodel_name="account.account.template",
+        string="Default Category Income Account",
+    )
+    account_expense_categ_id = fields.Many2one(
+        comodel_name="account.account.template",
+        string="Default Category Expense Account",
+    )
+
+    def install_chart_account(self):
+        """ install a chart of accounts for the given company """
+        user_company = self.env.user.company_id
+        self.env.user.company_id = self.new_company_id
+        self.sudo().chart_template_id.try_loading_for_current_company()
+        self.env.user.company_id = user_company
+
+    def create_bank_journals(self):
+        AccountJournal = self.env["account.journal"].sudo()
+        bank_journals = AccountJournal.search(
+            [("type", "=", "bank"), ("company_id", "=", self.new_company_id.id)]
+        )
+        vals = {
+            "type": "bank",
+            "company_id": self.new_company_id.id,
+        }
+        for i, bank in enumerate(self.bank_ids):
+            vals.update({"name": bank.acc_number, "bank_acc_number": bank.acc_number})
+            if i < len(bank_journals):
+                bank_journals[i].update(vals)
+            else:
+                vals.update(
+                    {
+                        "code": False,
+                        "sequence_id": False,
+                        "default_debit_account_id": False,
+                        "default_credit_account_id": False,
+                    }
+                )
+                AccountJournal.create(vals)
+
+    def create_sequences(self):
+        for sequence in self.sudo().sequence_ids:
+            sequence.copy({"company_id": self.new_company_id.id})
+
+    def create_company(self):
+        self.new_company_id = self.env["res.company"].create(
+            {"name": self.name, "user_ids": [(6, 0, self.user_ids.ids)]}
+        )
+        self.install_chart_account()
```

Comment:
```suggestion
        self.new_company_id = self.env["res.company"].create(
            {"name": self.name, "user_ids": [(6, 0, self.user_ids.ids)]}
        )
        self.new_company_id.chart_template_id = self.chart_template_id
        self.new_company_id.chart_template_id.try_loading_for_current_company()
```
It is necessary to apply this change or it does NOT apply the accounting plan correctly (if l10n_es is tested, this can be confirmed).